### PR TITLE
Pinning policy was changed to scoped pinning in enableSpeedbFeaturesCF

### PR DESCRIPTION
**Pull request by RoyBenMoshe**

### CODE REVIEW

**Comment by RoyBenMoshe** [comment]
This PR should be merged only after PR (#653 ) is merged

File: options/options.cc [hunk 1]
```diff
@@ -627,25 +628,15 @@ ColumnFamilyOptions* ColumnFamilyOptions::EnableSpeedbFeaturesCF(
...
-    cache_usage_options.options_overrides.insert(
-        {CacheEntryRole::kBlockBasedTableReader, role_options});
-    cache_usage_options.options_overrides.insert(
-        {CacheEntryRole::kCompressionDictionaryBuildingBuffer, role_options});
-    cache_usage_options.options_overrides.insert(
-        {CacheEntryRole::kFileMetadata, role_options});
+    std::ostringstream oss;
+    oss << "id=speedb_scoped_pinning_policy; capacity="
+        << 0.8 * shared_options.GetTotalRamSizeBytes()
+        /* << "; bottom_percent=10; mid_percent=80"*/;
```
Comment (udi-speedb) [inline options/options.cc:635]: The capacity for pinning should be 0.8 of the capacity we intend to allocate to pinning. That should be:
0.8 X (Total cache size - write buffer's manager size)
Total cache size should be using the actual configured size of the cache and not the total ram size the user has specified for the entire system IMO.
The above assumes we are using a WBM, and that it is configured to cost to cache.

**Comment by udi-speedb** [comment]
@RoyBenMoshe, @ayulas - I know I am not the reviewer, however, since I am working on static pinning, and making changes that should be in sync (e.g., #687), I briefly looked at the code and made the comment above.

**Comment by udi-speedb** [comment]
Closing as this was replaced by https://github.com/speedb-io/speedb/pull/715

**Pull request closed without merging**